specification/cost-management/resource-manager/readme.azureresourceschema.md

```diff
@@ -23,6 +23,19 @@ batch:
 
 Please also specify `--azureresourceschema-folder=<path to the root directory of your azure-resource-manager-schemas clone>`.
 
+### Tag: schema-costmanagement-2021-01-01 and azureresourceschema
+
+``` yaml $(tag) == 'schema-costmanagement-2021-01-01' && $(azureresourceschema)
+output-folder: $(azureresourceschema-folder)/schemas
+
+# all the input files in this apiVersion
+input-file:
+  - Microsoft.CostManagement/stable/2021-01-01/costmanagement.exports.json
+  - Microsoft.CostManagement/stable/2021-01-01/costmanagement.generatedetailedcostreport.json
+  - Microsoft.CostManagement/stable/2020-06-01/costmanagement.json
```

Comment:
Why do we need 2020-06-01 file referenced as an input for this new version 2021-01-01?

Reply:
I am following other RPs. I assume this will make available the latest API for all resource types.